app/models/user.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: DataTypes.INTEGER,
+      firstName: DataTypes.STRING,
+      lastName: DataTypes.STRING,
+      email: DataTypes.STRING,
+      password: DataTypes.STRING
+    },
+    {}
```

Comment:
I think this it's a case where the timestamp is important. Could you add timestamps?